packages/messages/src/browser/notifications.ts

```diff
@@ -72,8 +74,16 @@ export class Notifications {
                 button.addEventListener('click', () => {
```

Comment:
shouldn't we clear the timeout before `fn` call?

Reply:
Should not matter, but I will mve the clearTimeout before the close() call